invenio_vocabularies/ext.py

```diff
@@ -33,13 +37,31 @@ def init_app(self, app):
 
     def init_resource(self, app):
         """Initialize vocabulary resources."""
+        # Generic Vocabularies
         self.service = VocabulariesService(
             config=app.config["VOCABULARIES_SERVICE_CONFIG"],
         )
         self.resource = VocabulariesResource(
             service=self.service,
             config=app.config["VOCABULARIES_RESOURCE_CONFIG"],
         )
+        # Specific vocabularies
+        # - Affiliations
+        self.affiliations_service = AffiliationsService(
```

Comment:
major: I understand this is needed for the proxies (I am doubting now if we can actually have them...) but:
- Affiliations and subjects should not be registered in invenio-vocabularies. This step should happen in rdm-records. Think that they are "contrib", in this case we want both but there might be potential contribs that we do not want.
- It will break/duplicate with <https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/ext.py>
- If continuing with this, there should be a `init_services` method befor `init_resources`. (see link above)

Reply:
Correct. Everything in ``contrib`` is the responsibility of the module using it to register and provides proxies for, hence we cannot create and initialize them here.